design-proposals/klip-template.md

```diff
@@ -75,11 +84,8 @@ _Are we deprecating existing APIs with these changes? If so, when do we plan to
 
 _If we are removing old functionality, what is the migration plan?_
 
-## Performance Implications
```

Comment:
How come we're removing this section? I see the added note about performance under `What is in scope` but it still seems useful to call out performance as something to be considered with its own section.

Reply:
It's hard to actually write this down in any kind of useful way at the beginning of a design right? If you look at merged KLIPs, this is always left blank. 

I think it's more constructive to require an outline of the test plan for perf and scale, as well as outline expected performance characteristics in the 'inscope' section.

Reply:
Makes sense. LGTM!